runtime/compiler/codegen/CodeGenPhaseToPerform.hpp

```diff
@@ -55,6 +55,7 @@
     RegisterAssigningPhase,
     MapStackPhase,
     PeepholePhase,
+    ExpandInstructionsPhase,
```

Comment:
Will every architecture perform this phase then? (I know the initial motivation was Power).  If so, is there an early out on the pass if there is no work to perform?  I'd hate to enable a codegen phase on all architectures only to have most code generators spin their wheels eating up compile time.  If Power is the only adopter then perhaps it should be enabled in a Power-specific phase.

Reply:
There isn't a great precedence here.  I would like to keep the phase list the same between backends as much as possible until it really doesn't make sense to do so any longer for a particular backend (e.g., because it really must support a phase that only applies to that particular backend).  Hence, for this phase I'd prefer the `expandInstructions` function specialization just for Power approach.